api.md

```diff
@@ -266,71 +266,157 @@ Output:
 
 Status Code: 200 or 400
 
+### Create Project
+
+POST "/project_create"
```

Comment:
need to return project's id as well (otherwise frontend will never know the id)

-> we might need to add the invite code to the Project model & return it too in project_create as well & implement handlers for joining project